modules/ROOT/pages/whats-new-in-dw.adoc

```diff
@@ -0,0 +1,72 @@
+= What's New in DataWeave
+
+The 2.4 version of DataWeave introduces the following features:
+
+* Larger than memory strings.
+* New reader and writer properties for working with data formats.
```

Comment:
I think we should explicitly mention that XML now allows defining namespaces at the root level.

Reply:
```suggestion
* New reader and writer properties for working with data formats. For example, XML now enables you to define namespaces at the root level. 
```

Reply:
LGMT